kensu/pandas_gbq/pandas_gbq.py

```diff
@@ -1,10 +1,16 @@
+from kensu.utils.wrappers import remove_ksu_wrappers, remove_ksu_kwargs_wrappers
+
 from kensu.utils.kensu_provider import KensuProvider
 
 
 def wrap_pandas_gbq_write(method):
     def wrapper(*args, **kwargs):
         kensu = KensuProvider().instance()
-        df_result = method(*args, **kwargs)
+
+        new_args = remove_ksu_wrappers(args)
+        new_kwargs = remove_ksu_kwargs_wrappers(kwargs)
```

Comment:
@SammyEK i think it's quite common usecase - we could have a helper fn with remove wrappers from both args & kwargs.
and maybe it could call the `method` too (just then we should not get lost when explicit `self` is needed and when not).
no 3 lines -> 1 line